### DISCUSSION TITLE
How to get my copilot payment refund?

### DISCUSSION THREAD

**Discussion by Leginion** [community/community#40257] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I'm in Copilot trial and stoped use it after few days.
Copilot continued the period without any notice, and I totally forget that it will pay automatically after the trial.
I'm currently in a tough days, I want to cancel the order and get my money back.

I submitted the ticket to contact supports, but 6 hours past, no one response.
I was pointed here to look for help.

Thank you in advance.

**Comment by esQmo** [comment]
I guess you can opt out by going into your GitHub profile > Copilot or GitHub profile > Billing & Payments

**Reply by Leginion** [reply]
That actually stops the service, but can't get my money back. :(
Thank you for the advice. :)

**Comment by iamcjwilliams** [comment]
@LazyKnightX We're sorry for the trouble here. Our copilot team is currently experiencing a high volume of tickets, so responses will be delayed. We will respond as soon as possible. Thank you for your patience

**Comment by Akash1134** [comment, marked as answer]
Hi Everyone,

We now have the option to self-serve refunds for Copilot. Please head over to the [Help with your GitHub account](https://support.github.com/contact/account?flow=refund_copilot_individual) section and save yourself some time.

Note that, in most cases, the Virtual Agent may be able to offer an immediate refund. But, if your concern is not resolved, you can always use the contact form to get in touch with our billing team.
